CHANGELOG.md

```diff
@@ -1,10 +1,15 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [v1.0.0-rc.2] - TBD
+## [v1.0.0] - 2023-09-28
```

Comment:
Probably needs a links add and update at the bottom.

Reply:
fixed